app/api/Tickets/[id]/route.js

```diff
@@ -14,10 +14,22 @@ export async function PUT(req, { params }) {
 
     const body = await req.json();
     const ticketData = body.formData;
+    const ticketId = req.url.split("/").pop(); 
```

Comment:
we are already extracting the ticketId as a parameter of PUT function on line no 13

```javascript
export async function PUT(req, { params }) {
  try {
    const { id } = params;

    const body = await req.json();
```

so no need for this new ticketId object

Reply:
Done sir, made the required change.

Reply:
did you test the project locally after your latest commit?

it appears you removed the `ticketId` declaration in your latest commit but didn’t update the reference where it’s being called.


```javascript
const existingTicket = await Ticket.findById(ticketId);
```

 
 line no 17

Reply:
Sorry sir, made the changes.